packages/main/src/Table.js

```diff
@@ -574,6 +587,54 @@ class Table extends UI5Element {
 		}
 	}
 
+	_handleTab(event) {
+		const isNext = isTabNext(event);
+		const target = getNormalizedTarget(event.target);
+		const targetType = this.getFocusedElementType(event.target);
+
+		if (this.columnHeaderTabbables.includes(target)) {
+			if (isNext && this.columnHeaderLastElement === target) {
+				return this._focusNextElement(event);
+			}
+
+			return;
+		}
+
+		if (isNext && targetType === "columnHeader" && !this.columnHeaderTabbables.length) {
+			return this._focusNextElement(event);
+		}
+
+		if (targetType === "tableRow" || !targetType) {
+			return;
+		}
+
+		if (isNext) {
+			switch (targetType) {
+			case "tableGroupRow":
+				return this._focusNextElement(event);
+			case "moreButton":
+				return this._focusForwardElement(event, true);
+			}
+		} else {
+			switch (targetType) {
+			case "tableGroupRow":
+			case "columnHeader":
+				return this._focusForwardElement(event, false);
+			case "moreButton":
+				event.preventDefault();
+				return this.currentElement.focus();
+			}
+		}
```

Comment:
doesn't this look more readable? 

swithc(targetType) {
    case "tableGrowButton": 
              return isNext ? this._focsNextElement(..) : this._focusForwardElement(..)
    ....
}

Then you will not have to repeat the switch at all.